pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -226,10 +226,16 @@ def reset(self):
 
         @property
         def state(self):
-            """Copy the state vector data from the device to the host. A state vector Numpy array is explicitly allocated on the host to store and return the data."""
+            """Copy the state vector data from the device to the host. A state vector Numpy array is explicitly allocated on the host to store and return the data.
+            **Example**
+            >>> dev = qml.device('lightning.gpu', wires=1)
+            >>> dev.apply([qml.PauliX(wires=[0])])
+            >>> print(dev.state)
+            [0.+0.j 1.+0.j]
+            """
             state = np.zeros(1 << self.num_wires, dtype=self.C_DTYPE)
             state = self._asarray(state, dtype=self.C_DTYPE)
```

Comment:
Just one Q: Do we need `self._asarray` here? As Lightning-GPU device now inherits from `DefaultDevice`, this just calls `np.asarray` which I don't expect to do anything as `state` is already a numpy array.